Cohesivo rebranding (#3336)
ezsystems/developer-documentation: docs/product_catalog/quable/install_quable.md

php bin/console ibexa:quable:classification:sync
```

This command imports the product classification structure from [[= pim_product_name =]] into [[= product_name =]], ensuring that product categories are aligned.

!!! tip

>     To keep the classifications aligned, it is recommended that you run the `ibexa:quable:classification:sync` command every night, even when using synchronization with webhooks.

Reviewer: ```suggestion
    To keep the classifications aligned, it's recommended that you run the `ibexa:quable:classification:sync` command every night, even when using synchronization with webhooks.
```
